charts/pulsar/values.yaml

```diff
@@ -1193,6 +1193,77 @@ kube-prometheus-stack:
     enabled: true
   grafana:
     enabled: true
+    # Use random password at installation time for Grafana by default by setting empty value to `adminPassword`.
+    # You can find out the actual password by running the following command:
+    # kubectl get secret -l app.kubernetes.io/name=grafana -o=jsonpath="{.items[0].data.admin-password}" | base64 --decode
+    adminPassword:
+    # Configure Pulsar dashboards for Grafana
+    dashboardProviders:
+      dashboardproviders.yaml:
+        apiVersion: 1
+        providers:
+        - name: 'pulsar'
+          orgId: 1
+          folder: 'Pulsar'
+          type: file
+          disableDeletion: true
+          editable: true
+          options:
+            path: /var/lib/grafana/dashboards/pulsar
+    dashboards:
+      pulsar:
+        # Download the maintained dashboards from ASL 2.0 licenced repo https://github.com/streamnative/apache-pulsar-grafana-dashboard
+        bookkeeper:
```

Comment:
ASL 2.0 is not the correct abbreviation. ASL 1.0 and 1.1 was correct, but 2.0 is the Apache License and should be abbreviated AL 2.0

Reply:
Fixed in 65a5fc00